model/src/w3gridmd.F90

```diff
@@ -4949,14 +4949,7 @@ SUBROUTINE W3GRID()
           DEALLOCATE ( XGRDIN, YGRDIN )   
           CALL W3GNTX ( 1, 6, 6 )  
       ELSE 
-!
-!FA:  This distinction  between structured and unstructured
-! should be removed when XYB is replaced by XGRD and YGRD
-!
-        DO IX=1, NX
-          XGRD(:,IX) = XYB(IX,1)
-          YGRD(:,IX) = XYB(IX,2)
-          END DO
+!AR: This is already done in readmsh
```

Comment:
I'm a little confused by this comment.  Should it still be here?  If so, can you expand a little on this?

Reply:
This will be removed. Leftover from the merge.